models/mace/mace.yml

```diff
@@ -33,6 +33,7 @@ repo: https://github.com/ACEsuit/mace
 doi: https://doi.org/10.48550/arXiv.2401.00096
 paper: https://arxiv.org/abs/2401.00096
 requirements:
+  mace-torch: 0.3.0
```

Comment:
0.3.0 doesn't exist. https://pypi.org/project/mace-torch/0.3.3 is the earliest release